Source/ASVideoNode.mm

```diff
@@ -152,7 +152,8 @@ - (void)prepareToPlayAsset:(AVAsset *)asset withKeys:(NSArray<NSString *> *)requ
     NSError *error = nil;
     AVKeyValueStatus keyStatus = [asset statusOfValueForKey:key error:&error];
     if (keyStatus == AVKeyValueStatusFailed) {
-      NSLog(@"Asset loading failed with error: %@", error);
+        NSLog(@"Asset loading failed with error: %@", error);
+        [self.delegate videoNodeDidFailToInitAssetFor:self withError:error];
```

Comment:
Should we pass along the asset and maybe the key at hand? I imagine those objects might be helpful for investigations. Should be something like `-(void)videoNode:(ASVideoNode *)videoNode didFailToLoadValueForKey:(NSString *)key asset:(AVAsset *)asset error:(NSError *)error`.